src/common/src/hash/key.rs

```diff
@@ -164,16 +165,17 @@ pub trait HashKey:
 /// | (1, null) | (1, null) | true |
 /// | (null, 1) | (null, 1) | false |
 /// | (null, null) | (null, null) | false |
-pub struct JoinHashKey<'a, K> {
+#[derive(Clone)]
+pub struct JoinHashKey<K> {
     key: K,
-    null_matched: &'a FixedBitSet,
+    null_matched: Arc<FixedBitSet>,
```

Comment:
In most scenarios, the bitset is smaller than 1 byte, so just clone it directly without Arc is more efficient.

Reply:
Make sense.